k9mail/src/main/res/values/strings.xml

```diff
@@ -256,6 +256,10 @@ Please submit bug reports, contribute new features and ask questions at
     <string name="debug_enable_debug_logging_summary">Log extra diagnostic information</string>
     <string name="debug_enable_sensitive_logging_title">Log sensitive information</string>
     <string name="debug_enable_sensitive_logging_summary">May show passwords in logs.</string>
+    <string name="debug_use_condstore_extension_title">Use CONDSTORE</string>
```

Comment:
I think I would word this in a way that implies more heavily that something is being disabled that would usually be enabled. "Don't use CONDSTORE" / "Don't use CONDSTORE even if available"?